index.html

```diff
@@ -30,7 +33,7 @@
 
 <section id="abstract">
 
-This finding discusses the deprecation of third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies, the reasons for restricting them, and the challenges in removing them from the web platform. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
+This finding discusses why third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies cookies are harmful to the web, why they must be deprecated, and the challenges in removing them from the web platform. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```

Comment:
```suggestion
Third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies are harmful to the web, and must be removed from the web platform. This finding explains why they must be deprecated, and examines the challenges in removing them. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```

Reply:
I'm trying to punch this up a bit, make the statement stronger and easier for others to quote